apps/io-wallet-user-func/src/app/config.ts

```diff
@@ -50,6 +51,50 @@ export const ANDROID_CRL_URL =
 export const ANDROID_PLAY_INTEGRITY_URL =
   "https://www.googleapis.com/auth/playintegrity";
 
+export const MailConfig = t.type({
+  mailFeatureFlag: t.boolean,
+  mailSender: NonEmptyString,
+  mailupSecret: NonEmptyString,
+  mailupUsername: NonEmptyString,
+});
+
+export type MailConfig = t.TypeOf<typeof MailConfig>;
+
+export const WALLET_ACTIVATION_EMAIL_SUBJECT =
+  "IT Wallet - Aggiungi i tuoi documenti al Portafoglio di IO";
+export const WALLET_ACTIVATION_EMAIL_TEXT =
+  "IT Wallet - Aggiungi i tuoi documenti al Portafoglio di IO";
+export const WALLET_ACTIVATION_EMAIL_FAQ_LINK =
+  "https://io.italia.it/documenti-su-io/faq/#n1_12";
+export const WALLET_ACTIVATION_EMAIL_HANDLE_ACCESS_LINK =
+  "https://ioapp.it/it/accedi/?refresh=true";
+
+export const getMailConfigFromEnvironment: RE.ReaderEither<
+  NodeJS.ProcessEnv,
+  Error,
+  MailConfig
+> = pipe(
+  sequenceS(RE.Apply)({
+    mailFeatureFlag: pipe(
+      readFromEnvironment("MailFeatureFlag"),
+      RE.map(booleanFromString),
+      RE.orElse(() => RE.right(false)),
+    ),
+    mailSender: pipe(
+      readFromEnvironment("MailSender"),
+      RE.chainEitherKW(parse(NonEmptyString, "Invalid mail sender")),
+    ),
+    mailupSecret: pipe(
+      readFromEnvironment("MailupSecret"),
+      RE.chainEitherKW(parse(NonEmptyString, "Invalid mailup secret")),
+    ),
+    mailupUsername: pipe(
+      readFromEnvironment("MailupUsername"),
+      RE.chainEitherKW(parse(NonEmptyString, "Invalid mailup username")),
+    ),
```

Comment:
They should also be added to terraform

Reply:
What values ​​should these variables have in the `local.tf` file? Should the values ​​be the production ones? I ask because `MailupUsername` and `MailupSecret` are sensitive data

Reply:
keyvault should be used